Chore(stress-chaos): Add stress chaos docs (#481)
litmuschaos/litmus-docs: docs/pod-cpu-hog.md

## Details

- This experiment consumes the CPU resources on the application container on specified number of cores
- It simulates conditions where app pods experience CPU spikes either due to expected/undesired processes thereby testing how the overall application stack behaves when this occurs.
- Causes CPU resource consumption on specified application containers using cgroups and litmus nsutil which consume CPU resources of the given target containers.

> - It Can test the application's resilience to potential slowness/unavailability of some replicas due to high CPU load

Reviewer: ```suggestion
- It can test the application's resilience to potential slowness/unavailability of some replicas due to high CPU load
```

## Integrations

> - Pod CPU Hof can be effected using the chaos library: `litmus`

Reviewer: ```suggestion
- Pod CPU Hog can be affected using the chaos library: `litmus`
```

> - Pod CPU Hog can be affected using the chaos library: `litmus`

Reviewer 2: ```suggestion
- Pod CPU Hog can be effected using the chaos library: `litmus`
```

effected is the right word :)

  </tr>
  <tr>
    <td> LIB  </td>

>     <td> The chaos lib used to inject the chaos. Available libs are <code>litmus</code> and <code>litmus</code> </td>

Reviewer: ```suggestion
    <td> The chaos lib used to inject the chaos. Available libs are <code>litmus</code> and <code>pumba</code> </td>
```
